src/priv_file.rs

```diff
@@ -55,6 +55,22 @@ where
     }
 }
 
+/// Loads a log4rs logger configuration from a file.
+///
+/// Unlike `init_file`, this function does not initialize the logger; it only
+/// loads the `Config` and returns it.
+pub fn load_config_file<P>(path: P, deserializers: Deserializers) -> Result<Config, Error>
```

Comment:
Seems like we can make init_file use this to avoid duplication.

Reply:
`init_file` needs the `path`, `format`, `source`, and `config` variables. What are your thoughts on creating a helper method that returns all 4 of these variables, and `load_config_file` is simply a wrapper method that returns only `config`?

Reply:
Oh sure, I guess that might not work then.

Reply:
I guess the question is which would you rather have: more methods or a little duplication?

Reply:
I think this makes sense as-is!